tests/__init__.py

```diff
@@ -0,0 +1,56 @@
+'''This contains a a basetest case'''
+
+from unittest import TestCase
+import json
+# local imports
+try:
+    from app import create_app
+    from app.models import Database, Meal, User, Menu, Admin, Order
+except ModuleNotFoundError:
+    from .app import create_app
+    from .app.models import Database, Meal, User, Menu, Admin, Order
+
+
+class BaseTestClass(TestCase):
+    '''An abstract base class for tests, contains
+    all common variables methods'''
+
+    def setUp(self):
+        self.maxDiff = None
+        self.app = create_app('testing')
+        self.client = self.app.test_client()
+        self.new_user = {'username': 'john', 'email': 'john@mail.com',
+                         'password': 'password'}
+        self.admin_user = {'username': 'admin', 'email': 'admin@mail.com',
+                         'password': 'password', 'admin': True}
+        self.Database = Database()
+        self.Meal = Meal
+        self.Order = Order
+        self.Menu = Menu
+        self.Order = Order
+        self.User = User
+        self.Admin = Admin
+        self.meal = {'meal_id': 1, 'name': 'Fish', 'price': 100, 'description': 'Tasty Tilapia'}
+
+    def login_user(self):
```

Comment:
Why not prepend this test method with 'test_'?

Reply:
I meant to use it as a helper function. Since in test_apiendpoints some tests require the user to have logged in, I realized I'd have a lot of repeated code just to do logins so I decided to have a helper function for it in the base class.